gcloud/pubsub/subscription.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Define API Subscriptions."""
+
+from gcloud.exceptions import NotFound
+
+
+class Subscription(object):
+    """Subscriptions receive messages published to their topics.
+
+    See:
+    https://cloud.google.com/pubsub/reference/rest/v1beta2/projects/subscriptions
+
+    :type name: string
+    :param name: the name of the subscription
+
+    :type topic: :class:`gcloud.pubsub.topic.Topic`
+    :param topic: the topic to which the subscription belongs..
+
+    :type ack_deadline: int
+    :param ack_deadline: the deadline (in seconds) by which messages pulled
+                         from the back-end must be ACKed.
```

Comment:
ACKed -> acknowledged. (I suppose they could just click the link to learn more).